src/definitions/modules/video.js

```diff
@@ -85,20 +93,218 @@ $.fn.video = function(parameters) {
         },
 
         create: function() {
-          var
-            image = $module.data(metadata.image),
-            html = templates.video(image)
-          ;
-          $module.html(html);
-          module.refresh();
-          if(!image) {
-            module.play();
-          }
-          module.debug('Creating html for video element', html);
+          $video = element
+          module.debug('Reefrencing html for video element');
         },
-
+        
+        
+        control: {
+          
+          playpause: function($playpause) {
+            $playpause = $($playpause);
+            // from UI to video
+            $playpause.on('click' + eventNamespace, function() {
+              if(element.paused) {
+                element.play();
+              } else {
+                element.pause();
+              }
+            });
+            // from video to UI
+            $module
+              .on('play' + eventNamespace, function() {
+                $playpause.addClass('active')
+              }).on('pause' + eventNamespace, function() {
+                $playpause.removeClass('active')
+              });
+          },
+          
+          // see https://developer.mozilla.org/fr/docs/Web/API/HTMLMediaElement#playbackRate
+          // mode from 'switch', 'push'
+          playbackrate: function($playbackrate, rate, mode) {
+            $playbackrate = $($playbackrate);
+            // from UI to video
+            switch(mode) {
+              case 'push': default:
+                $playbackrate.on('mousedown' + eventNamespace, function() {
+                  element.playbackRate = rate;
+                });
+                $playbackrate.on('mouseup' + eventNamespace + ' mouseleave' + eventNamespace, function() {
```

Comment:
Don't use any anonymous functions, if it requires the `event` object use the `event: { }` naming scheme, see other modules, otherwise it should trigger an internal action, for example, clicking on playback rate should call something like `module.change.playbackRate`